src/templates/token.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Mjml,
+  MjmlBody,
+  MjmlColumn,
+  MjmlHead,
+  MjmlPreview,
+  MjmlSection,
+  MjmlStyle,
+  MjmlText,
+  MjmlTitle,
+  MjmlAttributes,
+  MjmlClass,
+  MjmlAll,
+  MjmlFont,
+  Mjml2HtmlOptions,
+} from 'mjml-react'
+
+import { render } from 'mjml-react'
+
+const colors = {
+  accent: '#fed823',
+  secondary: '#ff7bed',
+  formal: '#efecd6',
+  primaryDark: '#0d0030',
+}
+
+const generateTokenPage = ({ secret }: { secret: string }) => {
+  return (
+    <Mjml>
+      <MjmlHead>
+        <MjmlTitle>Here's your token!</MjmlTitle>
+        <MjmlPreview>Here's your token!</MjmlPreview>
+        <MjmlFont
+          name="Space Grotesk"
+          href="https://fonts.googleapis.com/css2?family=Space+Grotesk"
+        />
+        <MjmlFont
+          name="JetBrains Mono"
+          href="https://fonts.googleapis.com/css2?family=JetBrains+Mono"
+        />
+        <MjmlAttributes>
+          <MjmlClass name="text-accent" color={colors['accent']} />
+          <MjmlClass name="text-secondary" color={colors['secondary']} />
+          <MjmlClass name="text-formal-accent" color={colors['formal']} />
+          <MjmlClass
+            name="bg-secondary"
+            backgroundColor={colors['secondary']}
+          />
+          <MjmlClass
+            name="bg-primary-dark"
+            backgroundColor={colors['primaryDark']}
+          />
+          <MjmlClass
+            name="font-accent"
+            fontFamily="Space Grotesk, sans-serif"
+          />
+          <MjmlClass
+            name="font-primary"
+            fontFamily="JetBrains Mono, sans-serif"
+          />
+          <MjmlAll padding="0" />
+        </MjmlAttributes>
+        <MjmlStyle>{`
+            .body {
+              margin: 0;
+              padding: 0 20px;
+              -webkit-font-smoothing: antialiased;
+              -moz-osx-font-smoothing: grayscale;
+              background-color: #0d0030;
```

Comment:
Can we use variable here? Maybe css `--var()`?

Reply:
This is not supported by gmail https://www.caniemail.com/search/?s=var